pages/workspaces/[workspaceId]/settings.tsx

```diff
@@ -237,14 +244,6 @@ const WorkspaceSettings = ({ workspace, canDeleteWorkspace }: WorkspaceSettingsP
               placeholder="Workspace description"
               className="w-full md:w-3/4 max-w-lg"
             />
-            <div className="bg-white sticky-bottom fixed rounded-lg bottom-4 right-0 self-end m-6">
-              <Button
-                variant="primary"
-                className="z-50 flex gap-2.5 items-center cursor-pointer w-min sm:mt-0 self-end"
```

Comment:
The z-50 is nolonger required along with other z-indices removed in this PR. Undoes what as done in #2864.